raiwidgets/raiwidgets/_cohort.py

```diff
@@ -89,15 +113,34 @@ def __init__(self, method: str, arg: List[Any], column: str):
     def _validate_cohort_filter_parameters(
             self, method: str, arg: List[Any], column: str):
         """Validate the input values for the cohort filter.
-
         :param method: Cohort filter method from one of CohortFilterMethods.
         :type method: str
         :param arg: List of values to be used by the cohort filter.
         :type arg: list
         :param column: The column name from the dataset on which the filter
                        will be applied.
         :type column: str
+
+        The following validations can be performed on the cohort filter:-
+
+        1. Verify the correct types for method (expected string), column
+           (expected string) and arg (expected list).
+        2. The method value should be one of the filter string from
+           CohortFilterMethods.ALL.
+        3. The arg shouldn't be an empty list.
+        4. For all cohort filter methods in
+           CohortFilterMethods.SINGLE_VALUE_METHODS, the value in the arg
+           should be integer or float and there should be utmost one value
```

Comment:
"utmost one value"
Is this British English?  I'm more familiar with "at most", for example "at most one value", than the word "utmost".

Reply:
Changed to say only one value.